build.sbt

```diff
@@ -2,7 +2,7 @@ organization := "io.code-check"
 
 name := """github-api"""
 
-version := "0.2.1-SNAPSHOT"
+version := "0.3.0-SNAPSHOT"
```

Comment:
Update minor version since there is a risk that one of these dependencies could have breaking changes.